### PR TITLE
Update 'Panic Inc.' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.netlify.com/#!doc=%7B%22slug%22%3A%22panic%22%2C%22relevant-countries%22%3A%5B%22all%22%5D%2C%22categories%22%3A%5B%22entertainment%22%5D%2C%22name%22%3A%22Panic%20Inc.%22%2C%22runs%22%3A%5B%22Coda%22%2C%22Nova%22%2C%22Transmit%22%2C%22Prompt%22%2C%22Firewatch%22%2C%22Untitled%20Goose%20Game%22%2C%22Playdate%22%5D%2C%22address%22%3A%22315%20SW%2011th%20Ave%5Cn4th%20Floor%5CnPortland%2C%20OR%2097205%5CnUnited%20States%20of%20America%22%2C%22phone%22%3A%22%2B1%20503%20224%203451%22%2C%22email%22%3A%22gdpr%40panic.com%22%2C%22web%22%3A%22https%3A%2F%2Fpanic.com%2F%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fpanic.com%2Fprivacy%2F%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F957%22%5D%2C%22required-elements%22%3A%5B%7B%22desc%22%3A%22Email%22%2C%22type%22%3A%22name%22%2C%22optional%22%3Afalse%7D%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/panic.json [hunk 1]
```diff
@@ -8,15 +8,28 @@
...
     "name": "Panic Inc.",
     "runs": [
-        "Coda"
+        "Coda",
+        "Nova",
+        "Transmit",
+        "Prompt",
+        "Firewatch",
+        "Untitled Goose Game",
+        "Playdate"
```
Comment (fm) [inline companies/panic.json:17]: [Panic Sync](https://panic.com/sync) should also be added to this list

**Comment by mal-tee** [comment]
Someone should check if there are Privacy Policys for Untitled Goose Game and Firewatch. Maybe House House is responsible for UGG or Campo Santo/Valve for Firewatch.

**Comment by fm** [comment]
@mal-tee They don't have any, but took a look at the Panic Privacy Policy and it includes Untitled Goose Game in their scope. No mention of Firewatch, maybe keep UGG but remove Firewatch?

**Comment by mal-tee** [comment]
I can only find a reference to the UGG Shop, not the game itself. Or am I missing sth? :D

**Comment by fm** [comment]
Well, House House has no privacy policy... so Panic's is the only one available for UGG, even if it only mentions the shop. I've emailed House House to ask.

**Comment by mal-tee** [comment]
Oh, great. Thank you!

**Comment by fm** [comment]
The official answer is Panic

> The goose game doesn't do any kind of data collection, so we'd never thought it was necessary. 
>
> Panic is going to add Untitled Goose Game to their privacy policy for good measure, and state explicitly on there that there's no collection.

**Review by mal-tee — APPROVED** [review]
Thanks! 

They haven't updated the PP yet, but I don't want to leave this PR dangling indefinitely. If they update it, we can add UGG to `runs`

**Pull request merged**